sdk/search/search-documents/src/searchClient.ts

```diff
@@ -8,10 +8,12 @@ import {
   InternalPipelineOptions,
   createPipelineFromOptions,
   OperationOptions,
-  operationOptionsToRequestOptionsBase
+  operationOptionsToRequestOptionsBase,
+  RequestPolicyFactory,
+  bearerTokenAuthenticationPolicy
 } from "@azure/core-http";
```

Comment:
this reminds me - are we going to move search to corev2 soon?

Reply:
Yes. It is in our roadmap. We should be able to move it for the August release.